src/Abstractions/NexusMods.Abstractions.Loadouts/LoaderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using NexusMods.Abstractions.Loadouts.Ids;
+using NexusMods.Abstractions.Loadouts.Mods;
+using NexusMods.MnemonicDB.Abstractions;
+
+namespace NexusMods.Abstractions.Loadouts;
+
+/// <summary>
+/// Extensions to IDb/IConnection for loading loadouts and other entities
+/// </summary>
+public static class LoaderExtensions
```

Comment:
This should go into the `Extensions` folder and be named `DBExtensions` or similar. The `Extensions` namespace can also be removed from that folder, so all extensions should be available from just `NexusMods.Abstractions.Loadouts`.